src/Select.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { theme } from 'styled-system'
+
+import { Box, Flex, Icon, Label } from '../src'
+
+const parseChevronIcon = isExpanded => {
+  const ChevronIcon = isExpanded
+    ? Icon.extend`
+        transform: rotate(180deg);
+      `
+    : Icon
+
+  return <ChevronIcon name="chevronDown" size={12} color="black" />
+}
+
+const SelectFlex = Flex.extend`
+  cursor: pointer;
+  height: 48px;
+  border-radius: ${theme('radius')};
+  border-width: 1px;
+  border-style: solid;
+  border-color: ${theme('colors.borderGray')};
+`
+
+const SelectBox = Box.extend`
+  cursor: pointer;
+`
+
+const SelectLabel = Label.extend`
+  cursor: pointer;
+`
+
+const Select = props => (
```

Comment:
I think we should reserve the component name `Select` for the actual `<select>` element since React handles form elements differently and this has a completely different API from the HTML element – e.g. `onChange` `onFocus` `onBlur`

Reply:
I'd suggest that we call this component the `Dropdown` – since that's what it's commonly called – see https://getbootstrap.com/docs/4.0/components/dropdowns/

Reply:
Updated the component name to `Dropdown`.